app/views/orders/_cart_row.html.haml

```diff
@@ -1,7 +1,10 @@
-%tr
-  - if order_detail_iteration.first?
-    %td.centered{ rowspan: order_details.count }= link_to text("shared.remove"), remove_order_path(@order, order_detail), method: :put
-
+%tr{class: order_detail_iteration.first? ? "first-in-bundle" : ""}
+  %td
+    - if order_detail_iteration.first?
+      %strong
+        [#{link_to text("shared.remove"), remove_order_path(@order, order_detail), method: :put}]
+        - if order_detail.bundle
```

Comment:
I think with this here, I think we no longer need the bundle name tacked on before the product name. I think we can remove it from `product_description.html.haml`.